include/dlaf/common/index2d.h

```diff
@@ -223,10 +224,9 @@ Index2D<IndexT, Tag> computeCoordsRowMajor(std::ptrdiff_t linear_index,
                                            const Size2D<IndexT, Tag>& dims) noexcept {
   using dlaf::util::ptrdiff_t::mul;
 
-  DLAF_ASSERT_MODERATE(linear_index >= 0, "The linear index cannot be negative (",
-                       std::to_string(linear_index), ")");
-  DLAF_ASSERT_MODERATE(linear_index < mul(dims.rows(), dims.cols()), "Linear index ",
-                       std::to_string(linear_index), " does not fit into grid ", dims);
+  DLAF_ASSERT_MODERATE(linear_index >= 0, "The linear index cannot be negative!", linear_index);
```

Comment:
I wrapped `linear_index` with `std::to_string` because `uint8_t` or `int8_t` are interpreted as `char`s, so the equivalent ASCII mapping is printed instead of the numeric value (in case, see also next occurrences).

Reply:
I added a note to the code too.